api/src/main/java/org/openmrs/module/fhir2/providers/r3/ProcedureRequestFhirResourceProvider.java

```diff
@@ -54,6 +63,34 @@ public ProcedureRequest getProcedureRequestById(@IdParam @NotNull IdType id) {
 		return (ProcedureRequest) VersionConvertor_30_40.convertResource(serviceRequest, false);
 	}
 	
+	@Create
+	public MethodOutcome createProcedureRequest(@ResourceParam ProcedureRequest procedureRequest) {
+		return FhirProviderUtils.buildCreate(
+		    serviceRequestService.create((ServiceRequest) VersionConvertor_30_40.convertResource(procedureRequest, false)));
+	}
+	
+	@Update
+	public MethodOutcome updateProcedureRequest(@IdParam IdType id, @ResourceParam ProcedureRequest procedureRequest) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update");
+		}
+		
+		procedureRequest.setId(id.getIdPart());
+		
+		return FhirProviderUtils.buildUpdate(serviceRequestService.update(id.getIdPart(),
```

Comment:
This needs to be converted back into a `ProcedureRequest` before it's passed to `buildUpdate()`